ansible/host_vars/test-loadtest-01/version.yml

```
@@ -1,6 +1,6 @@
-SCHULCLOUD_SERVER_IMAGE_TAG: 27.4.0
-SCHULCLOUD_CLIENT_IMAGE_TAG: 27.4.1
-NUXT_CLIENT_IMAGE_TAG: 27.4.0
-SUPERHERO_DASHBOARD_IMAGE_TAG: 27.4.0
-SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.4.0
-ANTIVIRUS_CHECK_SERVICE_IMAGE_TAG: 2.0.2
+SCHULCLOUD_SERVER_IMAGE_TAG: 27.7.0
+SCHULCLOUD_CLIENT_IMAGE_TAG: 27.7.0
+NUXT_CLIENT_IMAGE_TAG: 27.7.0
+SUPERHERO_DASHBOARD_IMAGE_TAG: 27.6.0
+SCHULCLOUD_CALENDAR_IMAGE_TAG: 27.6.0
+ANTIVIRUS_CHECK_SERVICE_IMAGE_TAG: 2.0.3
```

Missing MESSENGER_SYNC_IMAGE_TAG
```suggestion
ANTIVIRUS_CHECK_SERVICE_IMAGE_TAG: 2.0.3
MESSENGER_SYNC_IMAGE_TAG: 2.2.0
```